Makefile.am

```diff
@@ -88,7 +88,8 @@ AM_CFLAGS += \
 	-Wundef \
 	-Wignored-qualifiers \
 	-Woverride-init \
-	-Wfloat-conversion
+	-Wfloat-conversion \
+        -Wbad-function-cast
```

Comment:
Please use tabs instead of spaces in Makefiles.

Reply:
Fixed 👍

Reply:
The first line is indented with 4 spaces, the second line is indented with 8 spaces.
Make sure that you use tab characters as your indentation as some editors may insert spaces instead of tabs depending on their configuration.

Reply:
The first line is indented with 4 spaces, the second line is indented with 8 spaces.
Make sure that you use tab characters as your indentation as some editors may insert spaces instead of tabs depending on their configuration.

Reply:
Yeah true it was the editor configuration issue